postgres/src/test/java/org/jdbi/v3/postgres/TestHStore.java

```diff
@@ -133,6 +134,7 @@ public void testWritesWithBindMap() {
     public interface CampaignDao {
 
         @SqlQuery("select caps from campaigns where id=:id")
+        @SingleValue
```

Comment:
I don't know whether this good thing or not... Probably good, if we know consider maps as containers.